substrate/frame/staking/src/benchmarking.rs

```diff
@@ -695,26 +800,26 @@ benchmarks! {
 		let stash = scenario.origin_stash1;
 
 		add_slashing_spans::<T>(&stash, s);
-		let l = StakingLedger::<T>::new(
-			stash.clone(),
-			asset::existential_deposit::<T>() - One::one(),
-		);
+		let l =
```

Comment:
The formatting here looks funny, did you run rustfmt? AFAIK benchmarks v1 was not compatible with rustfmt, so I expect a bunch of formatting issues but this line looks weird.

Reply:
You're right, using `v2` uncovered a few lints.

I ran `cargo +nightly-2024-09-11 fmt -p pallet-staking`, as you've guessed. This change is because the line would've exceeded 100 characters otherwise.